### ISSUE TITLE
Topic in latest ArXiV documents at 2022-03-01 20:24:45

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#523]
**Is your feature request related to a problem? Please describe.**
Topic in latest ArXiV documents at 2022-03-01 20:24:45
**Describe the solution you'd like**
* Topic 0: 
  * _Implementing simple spectral denoising for environmental audio  recordings_
  * _Parameterized Intractability for Multi-Winner Election under the  Chamberlin-Courant Rule and the Monroe Rule_
  * _Iterated Belief Change, Computationally_
  * _Automated Reasoning in Non-classical Logics in the TPTP World_
  * _Rethinking Immersive Virtual Reality and Empathy_
  * _Formulating Beurling LASSO for Source Separation via Proximal Gradient  Iteration_
  * _UnifiedQA-v2: Stronger Generalization via Broader Cross-Format Training_
  * _On change of measure inequalities for $f$-divergences_
  * _Information Extraction through AI techniques: The KIDs use case at  CONSOB_
  * _Measuring Trustworthiness or Automating Physiognomy? A Comment on Safra,  Chevallier, Grèzes, and Baumard (2020)_
* Topic 1: 
  * _Computational-Statistical Gaps in Reinforcement Learning_
  * _Between Stochastic and Adversarial Online Convex Optimization: Improved  Regret Bounds via Smoothness_
  * _Smoothed Online Learning is as Easy as Statistical Learning_
  * _Minimax Regret Optimization for Robust Machine Learning under  Distribution Shift_
  * _Online Control of Unknown Time-Varying Dynamical Systems_
  * _Bandit Learning with General Function Classes: Heteroscedastic Noise and  Variance-dependent Regret Bounds_
  * _Versatile Dueling Bandits: Best-of-both-World Analyses for Online  Learning from Preferences_
  * _Mirror Descent Strikes Again: Optimal Stochastic Convex Optimization  under Infinite Noise Variance_
  * _Improving Computational Complexity in Statistical Models with  Second-Order Information_
  * _Benign Underfitting of Stochastic Gradient Descent_
* Topic 2: 
  * _A QoE Model in Point Cloud Video Streaming_
  * _PCSCNet: Fast 3D Semantic Segmentation of LiDAR Point Cloud for  Autonomous Car using Point Convolution and Sparse Convolution Network_
  * _Patch-Based Deep Autoencoder for Point Cloud Geometry Compression_
  * _Joint Geometry and Color Projection-based Point Cloud Quality Metric_
  * _CG-SSD: Corner Guided Single Stage 3D Object Detection from LiDAR Point  Cloud_
  * _PointPCA: Point Cloud Objective Quality Assessment Using PCA-Based  Descriptors_
  * _On the impact of VR assessment on the Quality of Experience of Highly  Realistic Digital Humans_
  * _Variable Rate Compression for Raw 3D Point Clouds_
  * _Point Cloud Denoising via Momentum Ascent in Gradient Fields_
  * _Point Label Aware Superpixels for Multi-species Segmentation of  Underwater Imagery_
* Topic 3: 
  * _Unsupervised Representation Learning for Point Clouds: A Survey_
  * _RIConv++: Effective Rotation Invariant Convolutions for 3D Point Clouds  Deep Learning_
  * _Compressed Geometric Arrays for Point Cloud Processing_
  * _Graph Spring Network and Informative Anchor Selection for Session-based  Recommendation_
  * _ANTLER: Bayesian Nonlinear Tensor Learning and Modeler for Unstructured,  Varying-Size Point Cloud Data_
  * _Point Set Self-Embedding_
  * _Reliable Inlier Evaluation for Unsupervised Point Cloud Registration_
* Topic 4: 
  * _Soundify: Matching Sound Effects to Video_
  * _MANet: Improving Video Denoising with a Multi-Alignment Network_
  * _Unwinding Rotations Improves User Comfort with Immersive Telepresence  Robots_
  * _Punctuation Restoration_
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _FoV Privacy-aware VR Streaming_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _DCNGAN: A Deformable Convolutional-Based GAN with QP Adaptation for  Perceptual Quality Enhancement of Compressed Video_
  * _Nebula: Reliable Low-latency Video Transmission for Mobile Cloud Gaming_
* Topic 5: 
  * _(2.5+1)D Spatio-Temporal Scene Graphs for Video Question Answering_
  * _Malakai: Music That Adapts to the Shape of Emotions_
  * _The HaMSE Ontology: Using Semantic Technologies to support Music  Representation Interoperability and Musicological Analysis_
  * _ChMusic: A Traditional Chinese Music Dataset for Evaluation of  Instrument Recognition_
  * _Predicting emotion from music videos: exploring the relative  contribution of visual and auditory information to affective responses_
  * _MusIAC: An extensible generative framework for Music Infilling  Applications with multi-level Control_
  * _Music Sentiment Transfer_
  * _Actions Speak Louder than Listening: Evaluating Music Style Transfer  based on Editing Experience_
  * _S3T: Self-Supervised Pre-training with Swin Transformer for Music  Classification_
  * _Diversity in the Music Listening Experience: Insights from Focus Group  Interviews_
* Topic 6: 
  * _Cross-modal Music Emotion Recognition Using Composite Loss-based  Embeddings_
  * _Video Background Music Generation with Controllable Music Transformer_
  * _Unsupervised Learning of Deep Features for Music Segmentation_
  * _Enhancing Affective Representations of Music-Induced EEG through  Multimodal Supervision and latent Domain Adaptation_
  * _Learning long-term music representations via hierarchical contextual  constraints_
  * _Theme Transformer: Symbolic Music Generation with Theme-Conditioned  Transformer_
  * _Symbolic Music Loop Generation with VQ-VAE_
  * _MELONS: generating melody with long-term structure using transformers  and structure graph_
  * _Spectral embedding and the latent geometry of multipartite networks_
  * _System Safety and Artificial Intelligence_
* Topic 7: 
  * _Biased or Not?: The Story of Two Search Engines_
  * _Keyword Optimization in Sponsored Search Advertising: A Multi-Level  Computational Framework_
  * _Full-privacy secured search engine empowered by efficient genome-mapping  algorithms_
  * _Literature Review of the Pioneering Approaches in Cloud-based Search  Engines Powered by LETOR Techniques_
  * _Towards Building Economic Models of Conversational Search_
  * _Generating Clarifying Questions for Query Refinement in Source Code  Search_
  * _Improving BERT-based Query-by-Document Retrieval with Multi-Task  Optimization_
  * _Query Suggestion for Click-Absent Queries in Enterprise Search_
  * _Customising Ranking Models for Enterprise Search on Bilingual  Click-Through Dataset_
* Topic 8: 
  * _Domain-Augmented Domain Adaptation_
  * _Multi-Source Unsupervised Domain Adaptation via Pseudo Target Domain_
  * _Transferrable Contrastive Learning for Visual Domain Adaptation_
  * _Conditional Extreme Value Theory for Open Set Video Domain Adaptation_
  * _Attention-based Cross-Layer Domain Alignment for Unsupervised Domain  Adaptation_
  * _DGSS : Domain Generalized Semantic Segmentation using Iterative Style  Mining and Latent Representation Alignment_
  * _BioADAPT-MRC: Adversarial Learning-based Domain Adaptation Improves  Biomedical Machine Reading Comprehension Task_
  * _Deep learning based domain adaptation for mitochondria segmentation on  EM volumes_
* Topic 9: 
  * _Deep Learning, Natural Language Processing, and Explainable Artificial  Intelligence in the Biomedical Domain_
  * _A Smoothing and Thresholding Image Segmentation Framework with Weighted  Anisotropic-Isotropic Total Variation_
* Topic 10: 
  * _CAISE: Conversational Agent for Image Search and Editing_
  * _Towards Robust Neural Image Compression: Adversarial Attack and Model  Finetuning_
  * _Targeted Trojan-Horse Attacks on Language-based Image Retrieval_
  * _Confidence Calibration for Object Detection and Segmentation_
* Topic 11: 
  * _Improving Fake News Detection by Using an Entity-enhanced Framework to  Fuse Diverse Multimodal Clues_
  * _Why Do We Click: Visual Impression-aware News Recommendation_
  * _Local Gradient Hexa Pattern: A Descriptor for Face Recognition and  Retrieval_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
* Topic 12: 
  * _Sound Adversarial Audio-Visual Navigation_
  * _Amodal Panoptic Segmentation_
  * _Perceptual Consistency in Video Segmentation_
* Topic 13: 
  * _Robust Reinforcement Learning as a Stackelberg Game via  Adaptively-Regularized Adversarial Training_
  * _Towards Robust Stacked Capsule Autoencoder with Hybrid Adversarial  Training_
  * _Semi-Implicit Hybrid Gradient Methods with Application to Adversarial  Robustness_
  * _Using the left Gram matrix to cluster high dimensional data_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
* Topic 14: 
  * _Multi-Cue Adaptive Emotion Recognition Network_
* Topic 15: 
  * _Centre Symmetric Quadruple Pattern: A Novel Descriptor for Facial Image  Recognition and Retrieval_
  * _Cascaded Asymmetric Local Pattern: A Novel Descriptor for Unconstrained  Facial Image Recognition and Retrieval_
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
  * _Clustering by Hill-Climbing: Consistency Results_
  * _Visual Acoustic Matching_
  * _Strong Consistency for a Class of Adaptive Clustering Procedures_
  * _Optimal Clustering with Bandit Feedback_
  * _Mind Your Clever Neighbours: Unsupervised Person Re-identification via  Adaptive Clustering Relationship Modeling_
  * _Bio-Inspired Audio-Visual Cues Integration for Visual Attention  Prediction_
  * _Human Motion Detection Using Sharpened Dimensionality Reduction and  Clustering_
* Topic 16: 
  * _The Winning Solution to the iFLYTEK Challenge 2021 Cultivated Land  Extraction from High-Resolution Remote Sensing Image_
  * _ciscNet -- A Single-Branch Cell Instance Segmentation and Classification  Network_
  * _3DRM:Pair-wise relation module for 3D object detection_
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
* Topic 17: 
  * _Evaluation of an Audio-Video Multimodal Deepfake Dataset using Unimodal  and Multimodal Detectors_
  * _Deepfake Detection for Facial Images with Facemasks_
  * _Adversarial Example Devastation and Detection on Speech Recognition  System by Adding Random Noise_
  * _Multimedia Datasets for Anomaly Detection: A Survey_
  * _Exploring with Sticky Mittens: Reinforcement Learning with Expert  Interventions via Option Templates_
* Topic 18: 
* Topic 19: 
  * _On Pitfalls of Identifiability in Unsupervised Learning. A Note on:  "Desiderata for Representation Learning: A Causal Perspective"_
  * _Selection, Ignorability and Challenges With Causal Fairness_
  * _Diffusion Causal Models for Counterfactual Estimation_
  * _Learning Causal Overhypotheses through Exploration in Children and  Computational Models_
  * _Learning Latent Causal Dynamics_
  * _Relational Causal Models with Cycles:Representation and Reasoning_
  * _Effect Identification in Cluster Causal Diagrams_
  * _The Causal Marginal Polytope for Bounding Treatment Effects_
  * _Stochastic Causal Programming for Bounding Treatment Effects_
* Topic 20: 
  * _Unsupervised physics-informed disentanglement of multimodal data for  high-throughput scientific discovery_
  * _Rule-based Evolutionary Bayesian Learning_
  * _Robust Multi-Objective Bayesian Optimization Under Input Noise_
  * _Bayesian Robust Tensor Ring Model for Incomplete Multiway Data_
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
* Topic 21: 
  * _Large-scale Stochastic Optimization of NDCG Surrogates for Deep Learning  with Provable Convergence_
  * _Tackling benign nonconvexity with smoothing and stochastic gradients_
  * _The Power of Adaptivity in SGD: Self-Tuning Step Sizes with Unbounded  Gradients and Affine Variance_
  * _MSTGD:A Memory Stochastic sTratified Gradient Descent Method with an  Exponential Convergence Rate_
  * _On the influence of roundoff errors on the convergence of the gradient  descent method with low-precision floating-point computation_
  * _Interacting Contour Stochastic Gradient Langevin Dynamics_
  * _Meta-Learning for Simple Regret Minimization_
* Topic 22: 
* Topic 23: 
  * _SEC4SR: A Security Analysis Platform for Speaker Recognition_
  * _Evaluating the Adversarial Robustness of Adaptive Test-time Defenses_
* Topic 24: 
  * _FusionCount: Efficient Crowd Counting via Multiscale Feature Fusion_
  * _Hierarchical Interpretation of Neural Text Classification_
* Topic 25: 
  * _Robust Training under Label Noise by Over-parameterization_
  * _The Role of Reusable and Single-Use Side Information in Private  Information Retrieval_
* Topic 26: 
  * _Imitating Arbitrary Talking Style for Realistic Audio-DrivenTalking Face  Synthesis_
* Topic 27: 
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
  * _A Real-World Implementation of Unbiased Lift-based Bidding System_
  * _Temporal Subtyping of Alzheimer's Disease Using Medical Conditions  Preceding Alzheimer's Disease Onset in Electronic Health Records_
* Topic 28: 
* Topic 29: 
  * _Benchmarking Generative Latent Variable Models for Speech_
  * _Evaluating High-Order Predictive Distributions in Deep Learning_
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
* Topic 30: 
  * _FUNQUE: Fusion of Unified Quality Evaluators_
* Topic 31: 
  * _Sobolev Transport: A Scalable Metric for Probability Measures with Graph  Metrics_
  * _Reinforcing Local Feature Representation for Weakly-Supervised Dense  Crowd Counting_
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
* Topic 32: 
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
  * _Wisdom of (Binned) Crowds: A Bayesian Stratification Paradigm for Crowd  Counting_
* Topic 33: 
  * _FNR: A Similarity and Transformer-Based Approach to Detect Multi-Modal  Fake News in Social Media_
  * _Quality-aware News Recommendation_
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
* Topic 34: 
  * _Indiscriminate Poisoning Attacks on Unsupervised Contrastive Learning_
  * _Robotic Telekinesis: Learning a Robotic Hand Imitator by Watching Humans  on Youtube_
  * _Query Expansion and Entity Weighting for Query Reformulation Retrieval  in Voice Assistant Systems_
* Topic 35: 
  * _Large-scale Personalized Video Game Recommendation via Social-aware  Contextualized Graph Neural Network_
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Player Dominance Adjustment in Games_
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
* Topic 36: 
  * _Selective Credit Assignment_
  * _VLAD-VSA: Cross-Domain Face Presentation Attack Detection with  Vocabulary Separation and Adaptation_
* Topic 37: 
  * _Extending CLIP for Category-to-image Retrieval in E-commerce_
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
* Topic 38: 
* Topic 39: 
  * _A Complete Criterion for Value of Information in Soluble Influence  Diagrams_
* Topic 40: 
  * _Nonlinear Discrete Optimisation of Reversible Steganographic Coding_
* Topic 41: 
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
  * _Art Creation with Multi-Conditional StyleGANs_
* Topic 42: 
  * _Data Overlap: A Prerequisite For Disentanglement_
* Topic 43: 
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
* Topic 44: 
  * _How much depth information can radar infer and contribute_
  * _Light Robust Monocular Depth Estimation For Outdoor Environment Via  Monochrome And Color Camera Fusion_
  * _Utility and Feasibility of a Center Surround Event Camera_
  * _Multi-Objective Dual Simplex-Mesh Based Deformable Image Registration  for 3D Medical Images -- Proof of Concept_
* Topic 45: 
* Topic 46: 
  * _A Bibliometric Perspective of Social Science Scientific Communities of  Pakistan and India_
* Topic 47: 
* Topic 48: 
  * _Comfort and Sickness while Virtually Aboard an Autonomous Telepresence  Robot_
* Topic 49: 
  * _PQuAD: A Persian Question Answering Dataset_

**Describe alternatives you've considered**
None
**Additional context**
None

**Comment by wanted2** [comment]
This issue was created by an aggregate batch
